libgadget/init.c

```diff
@@ -96,6 +96,8 @@ set_init_params(ParameterSet * ps)
 
         All.BlackHoleOn = param_get_int(ps, "BlackHoleOn");
         All.WriteBlackHoleDetails = param_get_int(ps,"WriteBlackHoleDetails");
+        All.BH_DynamicalFriction = param_get_int(ps,"BH_DynamicalFriction");
+        All.BH_DFBoost = param_get_int(ps,"BH_DFBoost");
```

Comment:
Could you add a separate extra parameter that enables and disables repositioning? Something like BH_Reposition? That way we can easily disable repositioning without turning on dynamical friction.